### ISSUE TITLE
[DynamicPageTitle]: misalignment of default slot

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents#10160]
### Bug Description

The default slot is misaligned which becomes even more present if the [recommended font-sizes](https://sap.github.io/ui5-webcomponents/components/fiori/DynamicPageTitle/#heading) are used.

With recommended font-size:

<img width="902" alt="image" src="https://github.com/user-attachments/assets/706a8bc6-b08a-44ce-ae06-0c6f879ff346">

Without recommended font-size:

<img width="896" alt="image" src="https://github.com/user-attachments/assets/9b6e6fd0-aeca-48cb-b080-7eff41d85d08">

At least for the `ui5-title` component, the font-size is set internally, so it's redundant to set it from the app-side. As far as I know, this wasn't always the case, so it should still be possible to set it without breaking the layout.

### Affected Component

DynamicPageTitle

### Isolated Example

https://stackblitz.com/edit/ui5wc-dynamic-page-cnt3yy?file=index.html,main.js,package.json

### Steps to Reproduce

1. Open the[ StackBlitz example](https://stackblitz.com/edit/ui5wc-dynamic-page-cnt3yy?file=index.html,main.js,package.json)
2. See that the `Tag` is misaligned

### Priority

None

### UI5 Web Components Version

2.4.0

### Browser

Chrome

### Additional Context

Related public Slack message: https://openui5.slack.com/archives/CSQEJ2J04/p1730895222517669

### Organization

UI5WCR

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-p  ,
Could you please take a look at this? The issue can be reproduced here: https://stackblitz.com/edit/ui5wc-dynamic-page-cnt3yy?file=index.html,main.js,package.json
Regards,
Todor

**Comment by NakataCode** [comment]
Hello @Lukas742,

The alignment issue occurs because when the <ui5-title> element is not written as a one-liner, it automatically applies `white-space: pre-line`, which can cause unwanted line breaks or spacing. However, when we use the property `wrapping-type="None"`, it overrides the default wrapping behavior and ensures the title is aligned properly.

Does this suit your case? If you have any further questions, feel free to reach out.

Best regards,
Nikola

**Comment by Lukas742** [comment]
Hi @NakataCode 

> The alignment issue occurs because when the element is not written as a one-liner, 

Sorry, but to be honest I'm not really sure what you mean by that. Are you referring to the `ui5-title` element and what do you mean with one-liner? The only difference in the screenshots above is that I'm applying the recommended font-size in the first one, while omitting it in the second one.

Also, even with `wrapping-type="None"` applied to the `ui5-title`, the `ui5-tag` is still not correctly positioned, as there's a large space between it and the heading (size of subheading).
https://stackblitz.com/edit/ui5wc-dynamic-page-d87zjg?file=index.html,main.js,package.json

**Comment by NakataCode** [comment]
Hi @Lukas742

First of all, sorry for the confusion earlier—I misunderstood your point. I thought you were referring to the height alignment of the ui5-title with the KPI tag. However, I now understand that you're talking about the large space between the title and the KPI tag, not their height.

Regarding the large space, this issue will be fixed in the upcoming change: [PR #10163](https://github.com/SAP/ui5-webcomponents/pull/10163).

Also, to clarify the "one-liner" I mentioned: I meant that the title should be written in a single line, like this:
```js
<ui5-title slot="heading" style="font-size: var(--sapObjectHeader_Title_FontSize)">DynamicPage v2.4.0</ui5-title>
```

Thank you for your patience, and I hope this clears things up. Let me know if you have any further questions!

Best regards,

**Comment by Lukas742** [comment]
Hi @NakataCode 

> Regarding the large space, this issue will be fixed in the upcoming change: https://github.com/SAP/ui5-webcomponents/pull/10163.

Thanks :) 

> Also, to clarify the "one-liner" I mentioned: I meant that the title should be written in a single line, like this:

I took a closer look at this because I was surprised that code formatting could affect implementation details. It turns out that StackBlitz adds whitespace at the beginning and end of strings during formatting, which appears to be a bug in StackBlitz.

![image](https://github.com/user-attachments/assets/fd28ee2b-8b70-4d39-955b-0baca9ae33b5)
